src/watchmaker/workers/salt.py

```diff
@@ -255,6 +260,22 @@ def _build_salt_formula(self, extract_dir):
                 to_directory=extract_dir
             )
 
+        bundled_content = os.sep.join(
+            (static.__path__[0], 'salt', 'content')
+        )
+        if os.path.exists(bundled_content):
+            for subdir in next(os.walk(bundled_content))[1]:
+                if not subdir.startswith('.') and watchmaker.utils.copytree(
```

Comment:
Yeah, rewrite this so the `if` is just a test. Setup the test, and if the test passes then run `copytree()`.